bagisto/bagisto
# Fixed 8862

## Issue Reference
#8862 


## How To Test This?


- Go to 'Shop'
- Add a product to the cart and proceed to checkout.
- Fill in all the required fields and select the preferred shipping/payment method.
- Click on the "Place Order" button.
- Access the 'Admin Panel.'
- Open the order and generate a invoice.
- Refund it now.
- Check the mailbox of the customer and the admin.

## Documentation
